alerts/state.libsonnet

```diff
@@ -2,23 +2,25 @@
   prometheusAlerts+:: {
     groups+: [
       {
-        name: 'state-alert.rules',
+        name: 'cluster-state-alert.rules',
         rules: [
           {
-            alert: 'OCS_CephClusterErrorState',
+            alert: 'CephClusterErrorState',
             expr: |||
               ceph_health_status{%(cephExporterSelector)s} > 1
             ||| % $._config,
             'for': $._config.clusterStateAlertTime,
             labels: {
-              severity: 'critical',
+              severity: 'error',
```

Comment:
It is better to have severity as `Critical` and `Warning` because Alertmanager will use this as key to deduplicate, silence and route alerts.

Reply:
Yeah. Will update in next update.